codegen.yml

```diff
@@ -18,4 +18,4 @@ generates:
       withHooks: false
   ./graphql.schema.json:
     plugins:
-      - "introspection"
\ No newline at end of file
+      - "introspection"
```

Comment:
Please ignore. I'm not sure why GitHub diff is flagging this line as there is no change between the [before](https://github.com/mskcc/smile-dashboard/blob/master/codegen.yml) and [after](https://github.com/mskcc/smile-dashboard/blob/ec4fde4617c6a4600f16edcad584f89ff9ab684c/codegen.yml).